target/linux/qualcommax/files/arch/arm64/boot/dts/qcom/ipq6018-eap610-outdoor.dts

```diff
@@ -0,0 +1,148 @@
+// SPDX-License-Identifier: (GPL-2.0+)
```

Comment:
Mixing licenses makes maintenance quite more difficult. DTS and DTSI are used closely together. It's not like linking to some external library.

If we need to move any of your code to another `.dtsi` file that may raise licensing problems since your code uses more restrictive license than `.dtsi`.

Reply:
I see your arguments. I feel that they are speculative, and should not have bearing on the author's choice of licensing. I have submitted several GPL-2.0+ (no MIT) dts files to OpenWRT in the past without issue. Is the `qualcommax` target different?